Makefile

```diff
@@ -108,21 +106,24 @@ INSTALL_CORE_FILES = \
     stbt-screenshot \
     stbt-tv
 
-all: $(INSTALL_CORE_FILES)
+all: stbt.sh defaults.conf stbt/__init__.py $(INSTALL_CORE_FILES) \
```

Comment:
I'm probably just being blind, but where is the rule that generates `stbt/__init__.py`?